app/src/main/java/com/inveitix/android/clue/ui/MuseumDetailsActivity.java

```diff
@@ -59,12 +59,9 @@ protected void onCreate(Bundle savedInstanceState) {
         txtMuseumName.setText(museum.getName());
         txtMuseumInfo.setText(museum.getDescription());
         loadImage();
-        loadDwnBtn(museum);
+        loadDownloadBtn(museum);
     }
 
-    /**
-     * Animate Floating action button to appear and disappear when you starts and close the activity
-     */
     private void animateButton() {
```

Comment:
This documentation is useful:
Animate Floating action button to appear and disappear when you starts and close the activity